certified-connectors/Calendly/apiProperties.json

```diff
@@ -17,7 +17,9 @@
     },
     "iconBrandColor": "#006CFE",
     "scriptOperations": [
-      "CreateWebhookSubscription"
+      "CreateWebhookSubscription",
```

Comment:
Can you please update apiProperties.json as per the template here - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json